tests/meta/clean/conftest.py

```diff
@@ -234,3 +236,127 @@ def hk_sample_bundle_for_flow_cell_not_to_clean(
         "expires": timestamp_yesterday,
         "files": [],
     }
+
+
+@pytest.fixture
+def clean_retrieved_spring_files_api(
+    real_housekeeper_api: HousekeeperAPI,
+) -> CleanRetrievedSpringFilesAPI:
+    """Returns a CleanRetrievedSpringFilesAPI"""
+    return CleanRetrievedSpringFilesAPI(real_housekeeper_api)
+
+
+@pytest.fixture
+def path_to_old_retrieved_spring_file() -> str:
+    return Path("path", "to", "old", "retrieved", "spring", "file").as_posix()
+
+
+@pytest.fixture
+def path_to_newly_retrieved_spring_file() -> str:
+    return Path("path", "to", "newly", "retrieved", "spring", "file").as_posix()
+
+
+@pytest.fixture
+def path_to_archived_but_not_retrieved_spring_file() -> str:
+    return Path("path", "to", "archived", "spring", "file").as_posix()
+
+
+@pytest.fixture
+def path_to_fastq_file() -> str:
+    return Path("path", "to", "fastq", "file").as_posix()
+
+
+@pytest.fixture
+def paths_for_populated_clean_retrieved_spring_files_api(
+    path_to_old_retrieved_spring_file: str,
+    path_to_newly_retrieved_spring_file: str,
+    path_to_archived_but_not_retrieved_spring_file: str,
+    path_to_fastq_file: str,
+) -> list[str]:
+    return [
+        path_to_old_retrieved_spring_file,
+        path_to_newly_retrieved_spring_file,
+        path_to_archived_but_not_retrieved_spring_file,
+        path_to_fastq_file,
+    ]
+
+
+@pytest.fixture
+def retrieved_test_bundle_name() -> str:
+    return "retrieved_test_bundle"
+
+
+@pytest.fixture
+def archive_task_id() -> int:
+    return 123
+
+
+@pytest.fixture
+def retrieval_task_id() -> int:
+    return 321
+
+
+@pytest.fixture
+def archived_at_timestamp() -> datetime:
+    return datetime.now() - timedelta(weeks=52)
+
+
+@pytest.fixture
+def retrieved_at_old_timestamp() -> datetime:
+    return datetime.now() - timedelta(weeks=50)
+
+
+@pytest.fixture
+def retrieved_at_new_timestamp() -> datetime:
+    return datetime.now() - timedelta(days=1)
```

Comment:
Maybe check if the timestamps fixtures that we already have fullfill your time stamp needs

Reply:
Even if those timestamps contain the same value, I would want to wrap them or rename them since the names of the fixtures contain as much info as the actual date inside. If I can find some fixtures with scope=session, I might consider it, otherwise I prefer having these.

Reply:
I used some old ones!